### PR TITLE
Changed so that CLI supports specific extensions

### LINKED ISSUE #111: change CLI to support specific extenasions as does the Slang verifier
Support only:
- sl
- sl.yaml
- sl.yml

Do not support other extensions

### PR DESCRIPTION
**Pull request by tudorlesan**
Fixes #111

### CODE REVIEW

**Comment by tudorlesan** [comment]
fixes #111

File: score-lang-cli/src/main/java/org/openscore/lang/cli/utils/CompilerHelperImpl.java [hunk 1]
```diff
@@ -62,6 +63,16 @@ public CompilationArtifact compile(String filePath, List<String> dependencies) t
         Set<SlangSource> depsSources = new HashSet<>();
         File file = new File(filePath);
         Validate.isTrue(file.isFile(), "File: " + file.getName() + " was not found");
+        boolean validFileExtension = false;
+
+        Collection<File> slangFiles = FileUtils.listFiles(new File(file.getParent()), SLANG_FILE_EXTENSIONS, false);
+        for (File slangFile : slangFiles) {
+            if (StringUtils.equals(slangFile.getAbsolutePath(), file.getAbsolutePath())){
+                validFileExtension = true;
```
Comment (orius123) [inline score-lang-cli/src/main/java/org/openscore/lang/cli/utils/CompilerHelperImpl.java]: why do you need this validation?

**Comment by orius123** [comment]
tests?

File: score-lang-cli/src/main/java/org/openscore/lang/cli/utils/CompilerHelperImpl.java (hunk 1 above)
Comment (tudorlesan) [inline score-lang-cli/src/main/java/org/openscore/lang/cli/utils/CompilerHelperImpl.java]: I was suggested to use FileUtils.listFiles() which gets a collection of files with a specified list of extensions from a directory. In our case we specify a file, so I execute listFiles() for the file's parent directory. Afterwards I have to check that my file is among the list of files returned by the method.

Comment (orius123) [inline score-lang-cli/src/main/java/org/openscore/lang/cli/utils/CompilerHelperImpl.java]: and if the user specifies --cp?

Comment (tudorlesan) [inline score-lang-cli/src/main/java/org/openscore/lang/cli/utils/CompilerHelperImpl.java]: --cp is checked afterwards.The logic is as follows. First it checks that the source file exists. Then it gets all the files from the parent directory with our specified extensions and looks for the source file within them. Afterwards it gets all the files from the classpath with those extensions

**Pull request merged**